flask_rebar/rebar.py

```diff
@@ -547,12 +547,17 @@ def add_handler(
 
         # authenticators can be a list of Authenticators, a single Authenticator, USE_DEFAULT, or None
         authenticators_list: Sequence[Union[Type[USE_DEFAULT], Authenticator]] = []
-        if isinstance(authenticators, list):
-            authenticators_list = authenticators
+        if isinstance(authenticators, Sequence):
+            authenticators_list = list(authenticators)
```

Comment:
We're explicitly casting to a list here because we expect a list where this value is used?

Reply:
Honestly I am not really sure, in versions 3.1.1 and older, tuple was okay, but since 3.2.0 the check `isinstance(authenticators, list)` appeared for some reason. So I am casting to list here just to be sure (unsure if list is expected somewhere along the way or not)